FinancialTracker/ViewControllers/TransactionViewController.swift

```diff
@@ -80,6 +100,24 @@ class TransactionViewController: UIViewController {
             self.navigationController?.popViewController(animated: true)
         }
     }
+    
+    @objc func keyboardWillShow(notification: NSNotification) {
+        self.view.frame.origin.y = -50.0
```

Comment:
Better not use -50 directly but calculate the needed size dynamically based on the views that will actually get covered. At least try do so - the keyboard maybe different sizes on different devices.